ClosedXML/Excel/Drawings/IXLPicture.cs

```diff
@@ -8,6 +8,14 @@ public interface IXLPicture : IDisposable
     {
         IXLAddress BottomRightCellAddress { get; }
 
+        /// <summary>
+        /// Create a copy of the picture.
+        /// </summary>
+        /// <param name="targetSheet">A worksheet where put the copy to. When null the copy of the picture is created
```

Comment:
I don't think we should allow `targetSheet == null`. I would rather have a separate `IXLPicture.Duplicate()` method, which does the same.

Also update the parameter doctext to `The worksheet to which the picture will be copied.`